src/styles/blocks/card.scss

```
@@ -0,0 +1,94 @@
+.card {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+
+  width: 200px;
+  padding: 14px;
+  padding-top: 20px;
+
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  perspective: 100px;
+
+  &__image {
+    width: 160px;
+    height: 134px;
+    margin: 6px auto 42px;
+    transition: transform 400ms;
+    box-sizing: initial;
+    border: 3px solid transparent;
+
+    &:hover {
+      transform: translate3d(0,7px,17px);
+      border: 3px solid #f3f3f3;
+    }
+  }
+
+  &__content-text {
+    font-weight: 500;
+    font-style: normal;
+    font-size: 12px;
+    line-height: 15px;
+    margin-bottom: 6px;
+  }
+
+  &__code {
+    margin-bottom: 15px;
+  }
+
+  &__rating {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    margin-bottom: 20px;
+  }
+  &__stars {
+    display: flex;
+  }
+  .stars_star {
+    height: 15px;
+    width: 15px;
+    background: url("../images/star.svg") center no-repeat;
+  
+    &:not(:last-child) {
+      margin-right: 4px;
+      background: url("../images/star-active.svg") center no-repeat;
+    }
+  }
+  &__price {
+    display: flex;
+    justify-content: space-between;
+    align-items: baseline;
+    margin-bottom: 20px;
+  }
+
+  &__button {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    text-transform: uppercase;
+
+    text-decoration: none;
+    line-height: 16px;
+    font-size: 14px;
+    font-weight: 700;
+    min-height: 40px;
+    border: 1px solid $base--color--page;
+    color: #fff;
+    background-color: $base--color--page;
+    border-radius: 5px;
+    transition: 0.5s;
+  }
+
+  &__button:hover {
+    transition: 0.5s;
+    color: $base--color--page;
+    background-color: #fff;
+  }

```

you've already added transition to button itself

```suggestion
  &__button:hover {
    color: $base--color--page;
    background-color: #fff;
  }

```